tacoco-diagnoses

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+tacoco_home=$(pwd)
+
+ls -alh .
+ls -alh $tacoco_home/projects4testing/
+
+cd $tacoco_home/projects4testing/spiderMath_Gradle && gradle build
+cd $tacoco_home/projects4testing/spiderMath_JUnit4 && mvn clean compile test-compile -Dmaven.compiler.source=1.8 -Dmaven.compiler.target=1.8
+cd $tacoco_home/projects4testing/spiderMath_TestNG && mvn clean compile test-compile -Dmaven.compiler.source=1.8 -Dmaven.compiler.target=1.8
+
+cd $tacoco_home
+mvn exec:java -Plauncher -Dtacoco.sut=$(pwd)/projects4testing/spiderMath_Gradle -Dtacoco.home=$(pwd) -Dtacoco.project=spiderMath_Gradle -Danalyzer.opts="configs/tacoco-analyzer.config"
+
+if [ $? -ne 0 ];
+then
+    echo "Tacoco failed with spiderMath_Gradle..."
+    exit 1
+fi
+
+mvn exec:java -Plauncher -Dtacoco.sut=$(pwd)/projects4testing/spiderMath_TestNG -Dtacoco.home=$(pwd) -Dtacoco.project=spiderMath_TestNG -Danalyzer.opts="configs/tacoco-analyzer.config" 
+
+if [ $? -ne 0 ];
+then
+    echo "Tacoco failed with spiderMath_TestNG..."
+    exit 1
+fi
+
+mvn exec:java -Plauncher -Dtacoco.sut=$(pwd)/projects4testing/spiderMath_JUnit4 -Dtacoco.home=$(pwd) -Dtacoco.project=spiderMath_JUnit4 -Danalyzer.opts="configs/tacoco-analyzer.config"
+
+if [ $? -ne 0 ];
+then
+    echo "Tacoco failed with spiderMath_JUnit4..."
+    exit 1
+fi
+
```

Comment:
I thought this was pretty much self-contained in a TravisCI instance? so we should not have to worry about it getting added auto-magically to the repo? If we wanted to add this as a submodule (git) that would be a different conversation.

Reply:
I should have clarified. For travis this is okay, however, integration-test is also run locally when someone runs mvn install or mvn package. Without projects4testing this phase will fail.

Reply:
`mvn install -DskipTests -DskiptITs` // (assuming that i got the args right) this will skip both unit tests and the integration-test phase. Making this the default, and configuring it specifically for TravisCI should be a different change. Passing those flags on the command line in the meanwhile is not a terrible workaround.